tasks.go

```
@@ -56,6 +62,32 @@ func GetAllTasks(basepath string) ([]Task, error) {
 	return tasks, nil
 }
 
+// GetNagiosDataTask will return an array of Tasks each of which will dump the nagios data for one project.
+func GetNagiosTasks(projects []string, basepath string) ([]Task, error) {
+	var tasks []Task
+	for _, p := range projects {
+		pods, err := getResourceNamesBySubstr(p, "pod", "nagios")
+		if err != nil {
+			continue
```

Why ignore the error completely?